src/UI/components/map/layers/baseMapList.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import Collapse from '@mui/material/Collapse';
+import ExpandLess from '@mui/icons-material/ExpandLess';
+import ExpandMore from '@mui/icons-material/ExpandMore';
+import MapIcon from '@mui/icons-material/Map';
+import {ListButton} from './listButton';
+
+const BaseMapList= ({open, setOpen, openNestBasemapList, setOpenNestBasemapList,

```

Comment:
I need to check out the branch and look at this in more context, I'm trying to understand where these functions being passed in are coming from. Passing down lots of properties causes a lot of re-renders and aren't good for performance. I'd normally expect components to trigger actions that get processed by Redux and then other components watch for those changes rather than coupling components together by passing functions around.